.eslintrc.json

```diff
@@ -43,5 +43,18 @@
 		"react": {
 			"version": "detect"
 		}
-	}
+	},
+	"overrides": [
+		{
+		  "files": [
+			"**/*.test.js",
+			"**/*.test.jsx"
+		  ],
+		  "env": {
+			"jest": true
+		  }
+		}
+	  ]
+	
+
 }
```

Comment:
You may add a newline at the end of each file to remove some of the es lint errors

Reply:
oky am going to work on it